src/components/RecipeView.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Pagination from "./Pagination";
+import Recipe from "./Recipe";
+import "../styles/RecipeView.css";
+
+class RecipeView extends React.Component {
+
+  createRecipes = () => {
+    const recipes = this.props.data.map((recipe) => {
+      return <Recipe data={recipe} key={recipe.id}/>
+    })
+    return recipes;
+  }
+
+  render() {
+    return (
+      <div className="container">
+        {this.createRecipes()}
```

Comment:
i'd like a better name than createRecipes. maybe "populateRecipes"?

create in context of your application conflates itself with actual creation of recipes, which is confusing when you also have pagination going on.